src/js/in-content.js

```diff
@@ -8,7 +8,7 @@ function injectButton() {
 
   // Create our download button element
   const downloadButton = document.createElement('span');
-  downloadButton.innerHTML = `<a href=${downloadLink} target="_blank" class="_o4kklvw rc-VideoToolbarButton courseraDownloaderButton">Download in HD</a>`;
```

Comment:
Why we did this?

Reply:
basically it was these classes that were responsible for breakage of ui.